### PR TITLE
Added support for custom hosts in hipchat

### PR DESCRIPTION
**Pull request by chdi**
I added an additional optional parameter to use an custom hipChat-Host

### CODE REVIEW

File: lib/fastlane/actions/hipchat.rb [hunk 1]
```diff
@@ -25,6 +26,10 @@ def self.run(params)
           raise 'No HIPCHAT_API_VERSION given.'.red
         end
 
+        if api_host.nil?
```
Comment (KrauseFx) [inline lib/fastlane/actions/hipchat.rb]: How about

```ruby
api_host ||= 'api.hipchat.com'
```

**Comment by KrauseFx** [comment]
Thanks for the pull request. Please use a [config item](https://github.com/KrauseFx/fastlane/blob/master/lib/fastlane/actions/hipchat.rb#L85-L88) instead to provide default values and define the name of the environment variable.

**Comment by chdi** [comment]
Thanks for your advise. I changed it and moved to the master branch. Have a look here https://github.com/KrauseFx/fastlane/pull/370

**Comment by KrauseFx** [comment]
Closed in favour of https://github.com/KrauseFx/fastlane/pull/370, thanks again :+1:

**Pull request closed without merging**